easy-ui-react/src/Calendar/CalendarCell.module.scss

```diff
@@ -0,0 +1,57 @@
+@use "../styles/common" as *;
+
+.CellContainer {
+  padding: design-token("space.1.5") design-token("space.1");
+}
+
+.CalendarCell {
+  @include component-token("calendar-cell", "space", design-token("space.4-5"));
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: component-token("calendar-cell", "space");
+  height: component-token("calendar-cell", "space");
+  cursor: pointer;
+  border-radius: design-token("shape.border_radius.lg");
+  span {
+    color: design-token("color.primary.800");
+  }
+  :focus-visible {
```

Comment:
**question**: does this need to be `&:focus-visible` to target itself, or is the idea to target all descendants with `:focus-visible`?

Reply:
it should be `&:focus-visible`. thanks for catching this